whois-update/src/main/java/net/ripe/db/whois/update/handler/UpdateRequestHandler.java

```diff
@@ -178,11 +178,7 @@ private Collection<Update> attemptUpdatesOneByOne(final UpdateRequest updateRequ
     }
 
     private void processUpdateQueueBatchUpdate(final UpdateRequest updateRequest, final UpdateContext updateContext) {
-        try {
```

Comment:
Be very careful about changing this, it will have side-effects. I suggest logging the exception in the catch block for now but still swallow the exception, so we can find out when it happens.